src/components/TalentPage/Pagination.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react';
+import { Paginate } from './interfaces';
+
+const Pagination: FC<Paginate> = ({ filterPerPage, total, paginate }: Paginate) => {
+	const pageNumbers = [];
+	for (let i = 1; i <= Math.ceil(total / filterPerPage); i++) {
+		pageNumbers.push(i);
+	}
```

Comment:
its look like something wrong. why do you need it?

Reply:
It's need for dynamic showing page in pagination because total is change. And I need to calculate this.